CSC207-2022F-UofT/course-project-exam-centre
# Initial UI

What I have now
- Register / log in: `WelcomeDialog`
- View tests: `TestFrame`
- View solutions: `SolutionFrame`

What I don't have yet
- Discussion board
- Add course / test / solution
    - (individual components are there but not functional)

How to run the UI
1. Replace the file path with your own local file path to a PDF in `view/DocumentView.java`
2. Run `driver/MainTest.java`. Both "register / log in" and "view test" windows should open
3. On first build Gradle will download the ICEPDF API, might take some time
4. Only tested on Mac (macOS 12.6). Swing should be cross-platform but lmk if anything looks odd
